README.md

```diff
@@ -1,10 +1,70 @@
 # Drizzle Utils
 
-A library of frontend utilities for making dapps
+A library of tools for interacting with Ethereum smart contracts with a focus on [RxJS](https://github.com/ReactiveX/rxjs) streams so you can easily subscribe to events, contract state, and read-only methods.
 
-## For usage
+## Usage
 
-See the README for the core package [here](./packages/core/README.md).
+Note that the `$` at the end of a variable name denotes an [RxJS](https://github.com/ReactiveX/rxjs) stream. You can call `.subscribe` on it and use any RxJS stream operators of your choosing.
+
+### Getting started
+
+```js
+// import/require the packages you need
+const getWeb3 = require("@drizzle-utils/get-web3")
+const createDrizzleUtils = require("@drizzle-utils/core")
+
+// initialize the tooling
+const web3 = await getWeb3()
+const drizzleUtils = await createDrizzleUtils({ web3 })
+const accounts = await drizzleUtils.getAccounts()
+
+// `instance` is a web3 Contract instance of the deployed contract
+const instance = await drizzleUtils.getContractInstance({
+  artifact: contractArtifact
+})
+```
+
+### Streaming contract events
+
+Returns a Promise that resolves to an RxJS stream of events.
+
+```js
+const event$ = await drizzleUtils.createContractEvent$({
+  abi: contractArtifact.abi,
+  address: contractArtifact.networks[networkId].address
+})
+
+event$.subscribe(event => console.log(event))
+```
+
+### Streaming the contract state
+
+Returns a Promise that resolves to an RxJS stream of contract states.
+
+```js
+const state$ = await drizzleUtils.createContractState$({
+  artifact: contractArtifact,
+  provider: web3.currentProvider
+})
+
+state$.subscribe(state => console.log(state))
+```
+
+### Streaming a constant read-only method from a contract
+
+In this example, `get` is a read-only method in our contract.
+
+```js
+const call$ = await drizzleUtils.createContractCall$({
+  methodCall: instance.methods.get()
+})
+
+call$.subscribe(result => console.log(result))
+```
+
+### Using specific packages
+
+Refer to the specific package's README file for usage information.
 
 ## For development
```

Comment:
Is this section going to be filled out?

Reply:
I thought this was sufficient, but if not maybe I'll clarify.

Reply:
@honestbonsai I've updated the text here, is it better?

Reply:
@adrianmcli Sorry, I might not have seen the entire file, I was actually talking about `For development`, I dunno why but it looked empty to me earlier. But you do have something under there.